### ISSUE TITLE
Excessive AIR protection

### ISSUE THREAD

**Issue by SlimeDog** [dniym/IllegalStack#69]
Spigot 1.16.1 (jenkins 2830)
IllegalStack 2.1.7a
FastCraft 3.2.6

On opening FastCraft interface, very large number of these
```
[14:49:43] [Server thread/INFO]: [IllegalStack] -SlimeDog - AIR removed by protection Remove Overstacked Items, found on source Crafting Inventory
```
This did not occur prior to IllegalStack 2.1.7

**Comment by dniym** [comment]
Fixed in IllegalStack 2.1.7b

**Comment by SlimeDog** [comment]
Confirmed fixed in 2.1.7b
Thanks.
